go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -402,6 +402,15 @@ func (oc *DefaultNetworkController) Run(ctx context.Context) error {
 	klog.Infof("Starting all the Watchers...")
 	start := time.Now()
 
+	oc.wg.Add(1)
+	go func() {
+		defer oc.wg.Done()
+		oc.apbExternalRouteController.Run(1)
+	}()
+
+	// Wait to finish repairing the external gateway ECMP routes before resuming
+	oc.apbExternalRouteController.RepairWaitGroup.Wait()
```

Comment:
Any chance the repair could be run by itself outside of `apbExternalRouteController.Run`? I guess there is no issue to do it like its done here but it would be more straight forward an cleaner.